var/spack/repos/builtin/packages/pythia8/package.py

```diff
@@ -134,6 +134,11 @@ def setup_cxxstd(self):
             r"-std=c\+\+[0-9][0-9]", f"-std=c++{self.spec.variants['cxxstd'].value}", "configure"
         )
 
+    # Fix for https://gitlab.com/Pythia8/releases/-/issues/428
+    @when("@:8.311")
+    def patch(self):
+        filter_file(r"\|;n'", "|'", "configure")
```

Comment:
The regex looks quite generic. Please make it more targetted.

Maybe something like this?

UNTESTED!

```suggestion
        filter_file(r"[/]examples[/]Makefile[.]inc\|;n' \\", "/examples/Makefile.inc|' \\", "configure")
```